app/tsconfig.json

```diff
@@ -9,7 +9,6 @@
         "emitDecoratorMetadata": true,
         "experimentalDecorators": true,
         "sourceMap": true,
-        "noUnusedParameters": true,
```

Comment:
Any particular reason?

Reply:
It's in duplicate. The same entry shows up a few lines below.

Reply:
ah, gotcha